overlord/state/export_test.go

```diff
@@ -72,3 +72,7 @@ var (
 func (s *State) NumNotices() int {
 	return len(s.notices)
 }
+
+func (n *Notice) GetLastOccurred() time.Time {
```

Comment:
We should drop this method now that it's not needed, right?

Reply:
Ops!!! You are right. Fixing!